quasar/QuasarRAT
# Remote Desktop Protocol Basic Integration

Integration of a toggle to enable RDP on the client and forward connections on a special hostname (127.0.0.2) to the client.
This simply for now activates the default administrator account made by Windows and allows you to join it without any password (registry change for that as well).

To test it out:
1. Toggle so RDP connections are on (under Surveillance)
2. Start Proxifier or similar
3. Start the Reverse Proxy (on any port)
4. Add the reverse proxy to Proxifier
5. Add a rule for mstsc.exe (Remote Desktop Connections client)
6. Open up mstsc, put in 127.0.0.2 as host.
7. When asked for username, enter in "administrator" and no password.
8. Should be working.

More can be added for example a proper toggle with more packets and a nice form to allow users to add, remove, or edit user accounts on the client's machine.
Custom hostname could also be quite easily integrated or a list of appropriate hostnames (rewriting handler or similar for reverse proxy client?)

Food for thought, anyway :+1:
